services/provider/api/interfaces/interfaces.go

```diff
@@ -1,19 +1,22 @@
 package interfaces
 
 type StorageClientStatus interface {
-	// TODO: it was mistake not using full name of the field and we are just
-	// doing indirection for getters, change this interface after ensuring
-	// no client is using it
+	// these are indirected to getters generated by proto
 	GetPlatformVersion() string
 	GetOperatorVersion() string
+	GetOperatorNamespace() string
+
+	// these getters are generated by proto itself
```

Comment:
There is no need for that, this is an interface definition. We should not be mentioning implementation details